From 10413c522900d3aaadd5f5ae12163eee80ef18ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20M=C3=BCller?= <nexyon@gmail.com>
Date: Sun, 5 Mar 2017 12:22:00 +0100
Subject: [PATCH 01/16] Fix for seeking with modified pitch.

Check Blender bug report T50843: Pitched Audio renders incorrectly in VSE.
---
 src/devices/SoftwareDevice.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/devices/SoftwareDevice.cpp b/src/devices/SoftwareDevice.cpp
index 5fbb68c..c944b9e 100644
--- a/src/devices/SoftwareDevice.cpp
+++ b/src/devices/SoftwareDevice.cpp
@@ -357,6 +357,7 @@ bool SoftwareDevice::SoftwareHandle::seek(float position)
 	if(!m_status)
 		return false;
 
+	m_pitch->setPitch(m_user_pitch);
 	m_reader->seek((int)(position * m_reader->getSpecs().rate));
 
 	if(m_status == STATUS_STOPPED)
-- 
2.22.0

